.github/ISSUE_TEMPLATE/specification-issue.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Specification Issue
+about: Create a new issue related to the OpenAPI Specification
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**PLEASE NOTE:** This project only maintains the written [OpenAPI Specification](../blob/master/versions/3.0.2.md) and a few other supporting artifacts. No OpenAPI-related tooling lives here. If your issue is with a documentation format, UI, editor, programming framework, or other [OpenAPI implementation](../blob/master/IMPLEMENTATIONS.md), please post your issue in the appropriate project repository or contact the provider.
```

Comment:
Could we reword "documentation format", as this might dissuade people from submitting issues against the ReSpec (pretty) version of the spec?